app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players_data: list) -> int:
```

Comment:
Indicate that you expect to obtain a list of `Player` class instances for `players_data` in the type annotation